Library/ViewModels/DiscoveryNavigationHeaderViewModel.swift

```diff
@@ -297,6 +297,8 @@ private func stringsForTitle(params: DiscoveryParams) -> (filter: String, subcat
 
   if params.staffPicks == true {
     filterText = Strings.Projects_We_Love()
+  } else if params.hasLiveStreams == .some(true) {
+    filterText = "Kickstarter Live"
```

Comment:
Hey @awakinglim mind weighing in here? Curious what you think from brand, UX, localization perspective? This will appear in the discovery overlay and let you access cards for upcoming, live, and recent streams.